<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta http-equiv="X-UA-Compatible" content="IE=EmulateIE8" />
<link rel="stylesheet" type="text/css" href="css/allstyle.css"/>
<script type="text/javascript" src="js/jquery-3.4.1.min.js"></script>
<script type="text/javascript">
$(document).ready(function(){
  $(".1").mouseover(function(){
    $(".1").css("background-color","#506C96");
  });
  $(".1").mouseout(function(){
    $(".1").css("background-color","#FFFFFF");
  });
});
</script>
<title>映射</title>
<style type="text/css">
/*	原文*/

#apDiv4 {
    position: absolute;
    width: 56%;
    height: 600px;
    z-index: 0;
    left: 6px;
    top: 140px;
/*    border: 2px solid #506C96;*/
    float: left;
}
	
/*	评审意见*/
/*
	iframe{
	position: fixed;
    width: 40%;
    height: 600px;
    z-index: 0;
    left: 900px;
    top: 140px;
    float: right;
	}
*/
#apDiv5 {
    position: fixed;
    width: 40%;
/*    height: 1766px;*/
    z-index: 0;
    left: 900px;
    top: 140px;
    float: right;
}
	h1{
    font-size: 30px;
    text-align: left;
    font-family: cabin;
	}
	h2{
    font-size: 25px;
    text-align: left;
    font-family: cabin;
	}
	p{
    font-size: 20px;
    text-align: left;
    
	}
	h6{
    font-size: 20px;
    text-align: left;
    color: #006699;
    font-weight: 400;
    font-style: normal;
    font-family: gentium-basic;
	}
	.inform{
    font-size: 20px;
    text-align: left;
    color: #575757;
    font-weight: 400;
    font-style: normal;
    font-family: gentium-basic;
	}
	hr{
    margin-top: 10px;		
	}
</style>
<link href="SpryAssets/SpryTabbedPanels.css" rel="stylesheet" type="text/css" />
<script src="SpryAssets/SpryTabbedPanels.js" type="text/javascript"></script>
	

	
<!--The following script tag downloads a font from the Adobe Edge Web Fonts server for use within the web page. We recommend that you do not modify it.--><script>var __adobewebfontsappname__="dreamweaver"</script><script src="http://use.edgefonts.net/cabin:n4:default;abel:n4:default;anonymous-pro:n4:default;gentium-basic:n4:default.js" type="text/javascript"></script>
</head>

<body>
<div id="apDiv1">
  <div id="apDiv2">

	<ul class="top">
		<li><a href="login.html">log in</a></li>
		<li><a href="register.html">register</a></li>
		<li>search</li>
		<div class="icon-search"></div>
	</ul>
	
  </div>
  <div id="apDiv3">
	
	<ul class="nav">
	    <p class="name">NC 新视野</p>
		<li><a href="visualization.html">visualization</a></li>
		<li id="li1"><a href="#">subjects</a>
		    <ul id="ul1">
			    <li><a href="#"><a href="subject1-Physical sciences.html">Pysical sciences</a></a></li>
				<li><a href="#"><a href="subject2-Earth and environmental sciences.html">Earth and environmental sciences</a></a></li>
				<li><a href="#"><a href="subject3-Biological sciences.html">Biological sciences</a></a></li>
				<li><a href="#"><a href="subject4-Health sciences.html">Health sciences</a></a></li>
				<li><a href="#"><a href="subject5-Scientific community and society.html">Scientific community and society</a></a></li>
			</ul>
			<div class="tri"></div>
		</li>
		<li><a href="mapsearch.html">mapping</a></li>
	</ul>
	 <script>/* 显示二级菜单里栏JS鼠标效果 */
	  var oli=document.getElementById('li1');
	  var oul=document.getElementById('ul1');
      oli.onmouseover=function(){
			 oul.style.display='block';
		 }
	  oli.onmouseout=function(){
			 oul.style.display='none';
		 }
	 
	  </script> 
	  
	 
  </div>
</div>
	
	
<div id="apDiv4">
	<h6>Artical | Open Access | Published:19 February 2018</h6>
	<br size="0.5">
  <h1>Selection of GalNAc-conjugated siRNAs with limited off-target-driven rat hepatotoxicity</h1>
	<br>
	<h6>Maja M. Janas, Mark K. Schlegel, Carole E. Harbison, Vedat O. Yilmaz, Yongfeng Jiang, Rubina Parmar, Ivan Zlatev, Adam Castoreno, Huilei Xu, Svetlana Shulga-Morskaya, Kallanthottathil G. Rajeev, Muthiah Manoharan, Natalie D. Keirstead, Martin A. Maier & Vasant Jadhav </h6>
	<br>
	<h6 class="inform">Nature Communications volume 9, Article number: 723 (2018) | Cite this article <br>
  9762 Accesses | 72 Citations | 14 Altmetric | Metrics
	</h6>
	<br>
	<h2>Abstract</h2>
	<hr size="2">
	<p>Small interfering RNAs (siRNAs) conjugated to a trivalent N-acetylgalactosamine (GalNAc) ligand are being evaluated in investigational clinical studies for a variety of indications. The typical development candidate selection process includes evaluation of the most active compounds for toxicity in rats at pharmacologically exaggerated doses. The subset of GalNAc-siRNAs that show rat hepatotoxicity is not advanced to clinical development. Potential mechanisms of hepatotoxicity can be associated with the intracellular accumulation of oligonucleotides and their metabolites, RNA interference (RNAi)-mediated hybridization-based off-target effects, and/or perturbation of endogenous RNAi pathways. Here we show that rodent hepatotoxicity observed at supratherapeutic exposures can be largely attributed to RNAi-mediated off-target effects, but not chemical modifications or the perturbation of RNAi pathways. Furthermore, these off-target effects can be mitigated by modulating seed-pairing using a thermally destabilizing chemical modification, which significantly improves the safety profile of a GalNAc-siRNA in rat and may minimize the occurrence of hepatotoxic siRNAs across species.</p>
	<br>
	<h2>Introduction</h2>
	<hr>
	<p class="1">RNA interference (RNAi) is a highly conserved silencing pathway whereby short double-stranded RNAs downregulate expression of complementary target mRNAs to fine-tune gene expression and to protect against invasive nucleic acids1. These 20–25-nucleotide-long duplexes load into the RNA-induced silencing complex (RISC), which retains the antisense (guide) strand and discards the sense (passenger) strand2. There are two types of RNA-silencing triggers that can be loaded into cytoplasmic RISC in somatic cells: microRNAs (miRNAs) and siRNAs. miRNAs recognize target mRNAs via a partial sequence match predominantly within the 3′ untranslated region (3′UTR) leading to translational repression and mRNA destabilization, while siRNAs recognize target mRNAs via a full sequence match leading to mRNA cleavage1. Because of differences in the mechanism of action, siRNA-mediated gene-silencing activity is more potent than miRNA-mediated gene silencing. The core component of RISC is a member of the Argonaute (Ago) protein superfamily3. Although only Ago2 can catalyze siRNA-mediated mRNA cleavage4, all four Ago proteins (Ago1–4) participate in miRNA-mediated silencing, the major endogenous RNAi pathway in mammals.</p>
	<br>
	<p>This potent and selective natural RNAi pathway can be harnessed by exogenous siRNAs to knock down disease-causing mRNAs5. The liver is an attractive target organ for RNAi therapeutics owing to its association with a large number of human diseases combined with the possibility for efficient delivery. Conjugation to a trivalent N-acetylgalactosamine (GalNAc) ligand allows targeted delivery to hepatocytes via the abundant, hepatocyte-specific, and rapidly recycling asialoglycoprotein receptor6,7,8. To confer optimal drug-like properties (pharmacokinetics, pharmacodynamics, safety), we utilize siRNAs which are chemically modified. Typical modifications of the 2′-position of the ribose moiety include 2′-O-methyl (2′OMe) and 2′-deoxy-2′-fluoro (2′F), which, when combined with phosphorothioate (PS) modification at certain positions in the phosphodiester backbone8,9,10,11, have previously been reported to be well-tolerated in siRNAs in vitro8, 12, 13.</p>
	<br>
	<p>Our process of selecting GalNAc-siRNA conjugates for clinical development includes screening the most potent set of compounds in short-term rat toxicity studies at pharmacologically exaggerated exposures in a repeat-dose regimen. On average, we observe an approximate 40% attrition rate in these toxicity screens. We are generally able to identify both non-toxic and toxic siRNA sequences for the same target mRNA, arguing against on-target toxicity. The typical pathology findings that result in compound attrition are hepatocellular degeneration and necrosis with clinical pathology changes such as elevated alanine aminotransferase, aspartate aminotransferase, and total bilirubin. GalNAc-siRNAs that have an acceptable safety profile in these rat studies advance towards development candidate selection.</p>
	<br>
	<p>There are several potential mechanisms of hepatotoxicity to consider based upon the uptake pathway and pharmacologic activity of GalNAc-siRNAs. Mechanisms upstream of RISC loading could include class-wide toxicities associated with the accumulation of the endocytosed material in endosomes and lysosomes, or toxicities associated with chemical modifications of siRNAs or their metabolites. Endo-lysosomal accumulation of metabolically stable compounds could result in perturbation of endogenous endocytic processes by altering the structure and/or function of endocytic vesicles or proteins. Chemistry-related toxicities could result from non-specific binding to cellular proteins mediated by the chemical modifications used to protect siRNAs against nuclease digestion and immune recognition9, 10, leading to altered protein localization, structure, or function14, 15. Indeed, chemistry-related toxicities are sometimes observed with heavily modified oligonucleotide therapeutics both in vitro and in vivo. While this is especially true for single-stranded oligonucleotides with a high PS content that exhibit strong, non-sequence-specific protein binding14,15,16, these effects have not been reported to date with low PS content siRNAs12, 17.</p>
	<br>
	<p>Hepatotoxicity may also result from mechanisms downstream of RISC loading, including competition with the endogenous RNAi pathways and/or off-target repression of partially complementary mRNAs in a miRNA-like fashion. Since mammalian RISC is naturally occupied by miRNAs, it is conceivable that siRNAs could compete with miRNAs for RISC loading, resulting in miRNA destabilization and de-repression of their endogenous target mRNAs. Indeed, previous reports suggested that the major driver of small hairpin RNA hepatotoxicity in mice is perturbation of the most abundant hepatocyte-specific miRNA, miR-12218, 19. As described previously, siRNA-loaded RISC or antisense oligonucleotides may also bind and repress mRNAs with partial sequence complementarity20,21,22,23,24. Such RNAi-mediated off-target effects mimic the weak post-transcriptional silencing by endogenous miRNAs and are primarily driven by binding of the antisense seed region (nucleotides 2–8) to complementary sites in the 3′UTR25.</p>
	<br>
	<p>Here we describe a series of mechanistic studies demonstrating that RNAi-mediated, seed hybridization-based off-target effects are the major driver of hepatotoxicity observed with GalNAc-siRNAs in rodent toxicity screens. We find that blocking RISC loading of a random subset of hepatotoxic GalNAc-siRNAs without altering the PS, 2′OMe, or 2′F content mitigates hepatotoxicity while maintaining siRNA liver exposure, indicating little to no contribution of siRNA chemical modifications. Furthermore, blocking activity of the RISC-loaded antisense strand with complementary short oligonucleotides26, changing the sequence of the seed region without altering the PS, 2′OMe, or 2′F content, or utilizing a destabilizing glycol nucleic acid (GNA) nucleotide in the seed region also mitigate hepatotoxicity while maintaining comparable siRNA levels in both total liver and RISC. These results indicate that RISC loading is necessary but not sufficient for hepatotoxicity. Consistent with these data, global transcriptome profiling by RNAseq shows RISC loading-dependent downregulation of RNAs enriched for a seed match to the antisense but not the sense strand of GalNAc-siRNAs. Taken together, these studies implicate sequence-specific RNAi-based off-target effects, not a class effect based on siRNA chemistry or competition for RISC loading27, as the major driver of hepatotoxicity in rodents.</p>
	<br>
	<h2>Results</h2>
	<hr>
	<p>Blocking siRNA RISC loading mitigates hepatotoxicity
Efficient RISC loading and activity of small RNAi triggers depends on the presence of a monophosphate moiety at the 5′-end28, 29. While endogenous miRNAs naturally contain a 5′-monophosphate as a result of their biogenesis1, exogenous siRNAs are thought to be dependent on phosphorylation by kinases following intracellular uptake30, 31. To characterize the relationship of RISC loading to the hepatotoxicity observed with a random subset of modified GalNAc-siRNAs (Supplementary Fig. 1)8 in rodent toxicity studies, the 5′-end of the duplex antisense strand with previously established hepatotoxicity were capped (Fig. 1a) using three different types of nucleotide modifications designed to impede 5′-phosphorylation and thus RISC loading: 5′-inverted abasic (iB)32, 5′-deoxy-5′-(4-morpholinyl), or 5′-deoxy nucleotides (Parmar et al., manuscript in preparation). These capped siRNAs defective in RISC loading had the same PS, 2′OMe, and 2′F content as their RNAi-active counterparts that were identified in previous short-term repeat-dose rat toxicity screening studies as hepatotoxic and were designed against various target mRNAs with or without expected on-target activity in rodents.</p>
</div>
<!--		<iframe src="review.html" width="40%" height="100%"></iframe>-->
<div id="apDiv5" style=" overflow-y:auto; overflow-x:auto;  height:600px;">

  <div id="TabbedPanels1" class="TabbedPanels">
    <ul class="TabbedPanelsTabGroup">
      <li class="TabbedPanelsTab" tabindex="0">Round1</li>
      <li class="TabbedPanelsTab" tabindex="0">Round 2</li>
    </ul>
    <div class="TabbedPanelsContentGroup">
      <div class="TabbedPanelsContent">Reviewer #1 (Remarks to the Author) <br />
     
        In this work the authors identify a novel inhibitor of the splicing factor CLK kinases named T3 by 
        high-throughput screening of synthetic compounds. <p class="1">Through extensive and detailed        pharmacological assays, they thoroughly test the functionality of this inhibitor and demonstrate that T3 displays lower cytotoxicity and higher specificity than previous CLK inhibitors.</p> 
        To investigate the genome-wide impact of inhibition of CLK activity on transcription and splicing regulation, the authors carried out RNA-seq experiments using non-transformed immortalized cells and cancer cells treated with different doses of T3. These experiments revealed a widespread impact of CLK activity on alternative splicing regulation, with alternative exon cassette being the most regulated events. Moreover a significant overlap was found between T3-regulated splicing events and those regulated by CLK silencing, thus confirming the specificity of this drug.
        Interestingly, analysis of the RNA-seq datasets also revealed that T3-mediated inhibition of CLKs 
        induces the production of several conjoined gene transcripts. Consensus motifs for different RNA-
        binding proteins (RBPs) were identified within these transcripts and notably several of these RBPs  are found to interact with CLK2 by mass-spec analysis. These results suggest a previously unreported role for CLK kinases in transcript termination, whose inhibition can lead to aberrant expression of conjoined transcripts from flanking genes. Although the amount of data produced and described in this manuscript is impressive and of high quality, the overall message is somewhat lost due to excessive technical details. The amount of supplementary tables and figures is really overwhelming for the reader, and the manuscript appears more as a report of what was done rather than a report of a specific discovery.Description of the Figures is also sometimes confusing. For instance, different panels of Figure 3 are described in distant sections of the Results, and it is unclear why these results are shown in the same figure if the authors decided to describe them independently. A more consistent description of figures in the text would improve the clarity of the manuscript. Regarding the analysis of gene expression, splicing and conjoined transcripts, the authors remain very descriptive and do not attempt to identify mechanistic features. As an example, they do not search for the enrichment of consensus motifs for splicing factors regulated by CLKs in the exons/introns affected by T3. Similarly, although they search for motifs in the conjoined regions, they do not assess whether the RNA binding proteins potentially binding to these motifs are regulated by CLKs and/or are actually involved in the defect observed after treatment with T3. They also report detailed analysis of a number of clusters of splicing events, but the biological relevance of such clusters is not addressed. Lastly, the discussion appears as the summary of the description of the results,without providing additional insight to the reader. Overall, the suggestion is to limit the manuscript to one specific message and to go more in detail into its description. In particular, the role of CLKs in transcript termination and the induction of conjoined transcripts upon their inhibition appear the most novel and appealing findings. The detailed description of the compound identification and its biochemical characterization could be reported in a separate 
      </div>
   
      <div class="TabbedPanelsContent">内容 2</div>
	</div>
	</div>
</div>
<script type="text/javascript">
var TabbedPanels1 = new Spry.Widget.TabbedPanels("TabbedPanels1");
</script>

</body>
</html>
