<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
	<meta charset="UTF-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Paper Express</title>
	<link rel="stylesheet" type="text/css" href="fonts/feather/style.css">
	<link rel="stylesheet" type="text/css" href="css/demo.css" />
	<link rel="stylesheet" type="text/css" href="css/component.css" />
	<link href="css/bootstrap-4.4.1.css" rel="stylesheet" type="text/css">
<script src="js/modernizr.custom.js"></script>
	<script typet="text/javascript" src="http://libs.baidu.com/jquery/1.9.1/jquery.min.js"></script>
    <script>
        $(function () {            
            $(".like").click(function () {
                $(this).toggleClass('cs');                
            })
        })
    </script>
</head>
<body>
<div class="container">
<div style="text-align:center;clear:both;">
<script src="/gg_bd_ad_720x90.js" type="text/javascript"></script>
<script src="/follow.js" type="text/javascript"></script>
</div>
  <section class="slider">
		<div class="slide slide--current" data-content="content-1">
<div class="slide__mover">
  <h2 class="slide__title"><span></span>Selection of GalNAc-conjugated siRNAs with limited off-target-driven rat hepatotoxicity </h2>
<h2>&nbsp;<strong>Abstract</strong></h2>
              
        <div class="zoomer flex-center">
		  <blockquote>
<blockquote>
<h3 style="font-size: 130%"><strong>Small interfering RNAs (siRNAs) conjugated to a trivalent N-acetylgalactosamine (GalNAc) ligand are being evaluated in investigational clinical studies for a variety of indications. The typical development candidate selection process includes evaluation of the most active compounds for toxicity in rats at pharmacologically exaggerated doses. The subset of GalNAc-siRNAs that show rat hepatotoxicity is not advanced to clinical development. Potential mechanisms of hepatotoxicity can be associated with the intracellular accumulation of oligonucleotides and their metabolites, RNA interference (RNAi)-mediated hybridization-based off-target effects, and/or perturbation of endogenous RNAi pathways. Here we show that rodent hepatotoxicity observed at supratherapeutic exposures can be largely attributed to RNAi-mediated off-target effects, but not chemical modifications or the perturbation of RNAi pathways. Furthermore, these off-target effects can be mitigated by modulating seed-pairing using a thermally destabilizing chemical modification, which significantly improves the safety profile of a GalNAc-siRNA in rat and may minimize the occurrence of hepatotoxic siRNAs across species.</strong></h3>
		      <p class="like ">❤</p>
            </blockquote>
          </blockquote>
<div class="preview">
						
		    <div class="zoomer__area zoomer__area--size-2"></div>
		  </div>
</div>
</div>
			<h2 class="slide__title"><span></span></h2>
		</div>
		<div class="slide" data-content="content-2">
			<div class="slide__mover">
  <h2 class="slide__title" style="bottom: auto; left: 100px;"><span></span>Benchmarking microbiome transformations favors experimental quantitative approaches to address compositionality and sampling depth biases </h2>
<h2>&nbsp;<strong>Abstract</strong></h2>
              
        <div class="zoomer flex-center">
		  <blockquote>
		    <blockquote>
		      <h3 style="font-size: 130%"><strong>While metagenomic sequencing has become the tool of preference to study host-associated microbial communities, downstream analyses and clinical interpretation of microbiome data remains challenging due to the sparsity and compositionality of sequence matrices. Here, we evaluate both computational and experimental approaches proposed to mitigate the impact of these outstanding issues. Generating fecal metagenomes drawn from simulated microbial communities, we benchmark the performance of thirteen commonly used analytical approaches in terms of diversity estimation, identification of taxon-taxon associations, and assessment of taxon-metadata correlations under the challenge of varying microbial ecosystem loads. We find quantitative approaches including experimental procedures to incorporate microbial load variation in downstream analyses to perform significantly better than computational strategies designed to mitigate data compositionality and sparsity, not only improving the identification of true positive associations, but also reducing false positive detection. When analyzing simulated scenarios of low microbial load dysbiosis as observed in inflammatory pathologies, quantitative methods correcting for sampling depth show higher precision compared to uncorrected scaling. Overall, our findings advocate for a wider adoption of experimental quantitative approaches in microbiome research, yet also suggest preferred transformations for specific cases where determination of microbial load of samples is not feasible.</strong></h3>
				<p class="like ">❤</p>
            </blockquote>
          </blockquote>
<div class="preview">
						
		    <div class="zoomer__area zoomer__area--size-2"></div>
		  </div>
			
			  </div>
</div>
			<h2 class="slide__title"><span></span></h2>
		</div>
		<div class="slide" data-content="content-3">
			<div class="slide__mover">
  <h2 class="slide__title"><span></span>Chemoenzymatic modular assembly of O-GalNAc glycans for functional glycomics </h2>
<h2>&nbsp;<strong>Abstract</strong></h2>
              
        <div class="zoomer flex-center">
		  <blockquote>
		    <blockquote>
		      <h3 style="font-size: 130%"><strong>O-GalNAc glycans (or mucin O-glycans) play pivotal roles in diverse biological and pathological processes, including tumor growth and progression. Structurally defined O-GalNAc glycans are essential for functional studies but synthetic challenges and their inherent structural diversity and complexity have limited access to these compounds. Herein, we report an efficient and robust chemoenzymatic modular assembly (CEMA) strategy to construct structurally diverse O-GalNAc glycans. The key to this strategy is the convergent assembly of O-GalNAc cores 1–4 and 6 from three chemical building blocks, followed by enzymatic diversification of the cores by 13 well-tailored enzyme modules. A total of 83 O-GalNAc glycans presenting various natural glycan epitopes are obtained and used to generate a unique synthetic mucin O-glycan microarray. Binding specificities of glycan-binding proteins (GBPs) including plant lectins and selected anti-glycan antibodies towards these O-GalNAc glycans are revealed by this microarray, promoting their applicability in functional O-glycomics. Serum samples from colorectal cancer patients and healthy controls are assayed using the array reveal higher bindings towards less common cores 3, 4, and 6 than abundant cores 1 and 2, providing insights into O-GalNAc glycan structure-activity relationships.</strong></h3>
				<p class="like ">❤</p>
            </blockquote>
          </blockquote>
<div class="preview">
						
		    <div class="zoomer__area zoomer__area--size-2"></div>
		  </div>
			
			  </div>
</div>
			<h2 class="slide__title"><span></span></h2>
		</div>
	  <nav class="slider__nav">
			<button class="button button--nav-prev"><i class="icon icon--arrow-left"></i><span class="text-hidden">Previous paper</span></button>
			<button class="button button--zoom"><i class="icon icon--zoom"></i><span class="text-hidden">View details</span></button>
			<button class="button button--nav-next"><i class="icon icon--arrow-right"></i><span class="text-hidden">Next paper</span></button>
      </nav>
  
  </section>
	<section class="content">
		<div class="content__item" id="content-1">
	  </div>
		<div class="content__item" id="content-2">
		</div>
		<div class="content__item" id="content-3">
		</div>
		<button class="button button--close"><i class="icon icon--circle-cross"></i><span class="text-hidden">Close content</span></button>
  </section>
	<div style="text-align:center;clear:both;">
<script src="/gg_bd_ad_720x90-2.js" type="text/javascript"></script>
</div>
</div>

<script src="js/classie.js"></script>
<script src="js/dynamics.min.js"></script>
<script src="js/main.js"></script>

</body>
</html>