<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
	<meta charset="UTF-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Paper Express Biological</title>
	<link rel="stylesheet" type="text/css" href="fonts/feather/style.css">
	<link rel="stylesheet" type="text/css" href="css/demo.css" />
	<link rel="stylesheet" type="text/css" href="css/component.css" />
	<link href="css/bootstrap-4.4.1.css" rel="stylesheet" type="text/css">
<script src="js/modernizr.custom.js"></script>
	<script typet="text/javascript" src="http://libs.baidu.com/jquery/1.9.1/jquery.min.js"></script>
    <script>
        $(function () {            
            $(".like").click(function () {
                $(this).toggleClass('cs');                
            })
        })
    </script>
</head>
<body>
<div class="container">
<div style="text-align:center;clear:both;">
<script src="/gg_bd_ad_720x90.js" type="text/javascript"></script>
<script src="/follow.js" type="text/javascript"></script>
</div>
  <section class="slider">
		<div class="slide slide--current" data-content="content-1">
<div class="slide__mover">
  <h2 class="slide__title"><span></span>A catalog of the diversity and ubiquity of bacterial microcompartments </h2>
<h2>&nbsp;<strong>Abstract</strong></h2>
              
        <div class="zoomer flex-center">
		  <blockquote>
<blockquote>
<h3 style="font-size: 130%"><strong>Bacterial microcompartments (BMCs) are organelles that segregate segments of metabolic pathways which are incompatible with surrounding metabolism. BMCs consist of a selectively permeable shell, composed of three types of structurally conserved proteins, together with sequestered enzymes that vary among functionally distinct BMCs. Genes encoding shell proteins are typically clustered with those for the encapsulated enzymes. Here, we report that the number of identifiable BMC loci has increased twenty-fold since the last comprehensive census of 2014, and the number of distinct BMC types has doubled. The new BMC types expand the range of compartmentalized catalysis and suggest that there is more BMC biochemistry yet to be discovered. Our comprehensive catalog of BMCs provides a framework for their identification, correlation with bacterial niche adaptation, experimental characterization, and development of BMC-based nanoarchitectures for biomedical and bioengineering applications.</strong></h3>
		      <p class="like ">❤</p>
            </blockquote>
          </blockquote>
<div class="preview">
						
		    <div class="zoomer__area zoomer__area--size-2"></div>
		  </div>
</div>
</div>
			<h2 class="slide__title"><span></span></h2>
		</div>
		<div class="slide" data-content="content-2">
			<div class="slide__mover">
  <h2 class="slide__title" style="bottom: auto; left: 100px;"><span></span>Systematic detection of functional proteoform groups from bottom-up proteomic datasets </h2>
<h2>&nbsp;<strong>Abstract</strong></h2>
              
        <div class="zoomer flex-center">
		  <blockquote>
		    <blockquote>
		      <h3 style="font-size: 130%"><strong>To a large extent functional diversity in cells is achieved by the expansion of molecular complexity beyond that of the coding genome. Various processes create multiple distinct but related proteins per coding gene – so-called proteoforms – that expand the functional capacity of a cell. Evaluating proteoforms from classical bottom-up proteomics datasets, where peptides instead of intact proteoforms are measured, has remained difficult. Here we present COPF, a tool for COrrelation-based functional ProteoForm assessment in bottom-up proteomics data. It leverages the concept of peptide correlation analysis to systematically assign peptides to co-varying proteoform groups. We show applications of COPF to protein complex co-fractionation data as well as to more typical protein abundance vs. sample data matrices, demonstrating the systematic detection of assembly- and tissue-specific proteoform groups, respectively, in either dataset. We envision that the presented approach lays the foundation for a systematic assessment of proteoforms and their functional implications directly from bottom-up proteomic datasets.</strong></h3>
				<p class="like ">❤</p>
            </blockquote>
          </blockquote>
<div class="preview">
						
		    <div class="zoomer__area zoomer__area--size-2"></div>
		  </div>
			
			  </div>
</div>
			<h2 class="slide__title"><span></span></h2>
		</div>
		<div class="slide" data-content="content-3">
			<div class="slide__mover">
  <h2 class="slide__title"><span></span>Composition and stage dynamics of mitochondrial complexes in Plasmodium falciparum </h2>
<h2>&nbsp;<strong>Abstract</strong></h2>
              
        <div class="zoomer flex-center">
		  <blockquote>
		    <blockquote>
		      <h3 style="font-size: 130%"><strong>Our current understanding of mitochondrial functioning is largely restricted to traditional model organisms, which only represent a fraction of eukaryotic diversity. The unusual mitochondrion of malaria parasites is a validated drug target but remains poorly understood. Here, we apply complexome profiling to map the inventory of protein complexes across the pathogenic asexual blood stages and the transmissible gametocyte stages of Plasmodium falciparum. We identify remarkably divergent composition and clade-specific additions of all respiratory chain complexes. Furthermore, we show that respiratory chain complex components and linked metabolic pathways are up to 40-fold more prevalent in gametocytes, while glycolytic enzymes are substantially reduced. Underlining this functional switch, we find that cristae are exclusively present in gametocytes. Leveraging these divergent properties and stage dynamics for drug development presents an attractive opportunity to discover novel classes of antimalarials and increase our repertoire of gametocytocidal drugs.</strong></h3>
				<p class="like ">❤</p>
            </blockquote>
          </blockquote>
<div class="preview">
						
		    <div class="zoomer__area zoomer__area--size-2"></div>
		  </div>
			
			  </div>
</div>
			<h2 class="slide__title"><span></span></h2>
		</div>
	  <nav class="slider__nav">
			<button class="button button--nav-prev"><i class="icon icon--arrow-left"></i><span class="text-hidden">Previous paper</span></button>
			<button class="button button--zoom"><i class="icon icon--zoom"></i><span class="text-hidden">View details</span></button>
			<button class="button button--nav-next"><i class="icon icon--arrow-right"></i><span class="text-hidden">Next paper</span></button>
      </nav>
  
  </section>
	<section class="content">
		<div class="content__item" id="content-1">
	  </div>
		<div class="content__item" id="content-2">
		</div>
		<div class="content__item" id="content-3">
		</div>
		<button class="button button--close"><i class="icon icon--circle-cross"></i><span class="text-hidden">Close content</span></button>
  </section>
	<div style="text-align:center;clear:both;">
<script src="/gg_bd_ad_720x90-2.js" type="text/javascript"></script>
</div>
</div>

<script src="js/classie.js"></script>
<script src="js/dynamics.min.js"></script>
<script src="js/main.js"></script>

</body>
</html>